ansible/roles/analytics-druid/defaults/main.yml

```diff
@@ -33,7 +33,7 @@ druid_request_logging_type: "file"
 
 druid_storage_type: "azure"
 
-druid_extensions_list : '"druid-azure-extensions", "graphite-emitter", "postgresql-metadata-storage", "druid-kafka-indexing-service"'
+druid_extensions_list : '"druid-azure-extensions", "graphite-emitter", "postgresql-metadata-storage", "druid-kafka-indexing-service", "druid-datasketches"'
```

Comment:
This change is already merged to base branch, why is this showing up in this PR?